src/CodeGen_LLVM.cpp

```diff
@@ -978,7 +973,7 @@ llvm::Function *CodeGen_LLVM::add_argv_wrapper(llvm::Function *fn,
     std::vector<llvm::Value *> wrapper_args;
     for (llvm::Function::arg_iterator i = fn->arg_begin(); i != fn->arg_end(); i++) {
         // Get the address of the nth argument
-        llvm::Value *ptr = CreateConstGEP1_32(builder, i8_t->getPointerTo(),
+        llvm::Value *ptr = CreateConstGEP1_32(builder.get(), i8_t->getPointerTo(),
```

Comment:
I'm surprised the `.get()` is necessary, doesn't a unique_ptr have an implicit conversion?

Reply:
Apparently not. I get compiler errors without it.